react/features/base/media/components/web/AudioTrack.js

```diff
@@ -193,10 +196,58 @@ export default class AudioTrack extends Component<Props> {
      */
     _detachTrack(track) {
         if (this._ref && track && track.jitsiTrack) {
+            clearTimeout(this._playTimeout);
+            this._playTimeout = undefined;
             track.jitsiTrack.detach(this._ref);
         }
     }
 
+    _play: ?number => void;
+
+    /**
+     * Plays the uderlying HTMLAudioElement.
+     *
+     * @param {number} retries - The number of previously failed retries.
+     * @returns {void}
+     */
+    _play(retries = 0) {
+        if (!this._ref) {
+            // nothing to play.
+
+            return;
+        }
+        const { autoPlay, id } = this.props;
+
+        if (autoPlay) {
+            // Ensure the audio gets play() called on it. This may be necessary in the
+            // case where the local video container was moved and re-attached, in which
+            // case the audio may not autoplay.
+            this._ref.play()
+            .then(() => {
+                if (retries !== 0) {
+                    // success after some failures
+                    this._playTimeout = undefined;
+                    sendAnalytics(createAudioPlaySuccessEvent(id));
+                    logger.info(`Successfully played audio track! retries: ${retries}`);
+                }
+            })
+            .catch(e => {
```

Comment:
`.then().catch()`  is not the same as `.then(onSuccess, onError)`. This code will treat any exceptions inside the `onSuccess` callback as playback error

Reply:
Nice catch! Will fix it!

Reply:
fixed in https://github.com/jitsi/jitsi-meet/pull/9271